blogs.json

```diff
@@ -5339,6 +5339,12 @@
             "site_url": "https://code.allaboutapps.at/categories/ios/",
             "feed_url": "https://code.allaboutapps.at/categories/ios/index.xml"
           },
+          {
+            "title": "Appcircle Blog",
+            "author": "Appcircle",
+            "site_url": "https://appcircle.io/blog",
```

Comment:
I'm afraid the schema needs a valid RSS feed to be entered here under `feed_url`. This site is primarily used by RSS readers and other software to automatically digest the articles via all of the RSS feeds and so the `feed_url` is mandatory.

I couldn't find an RSS feed for your site. Do you have one?